ecogvis/resources/example_metafile.yml

```diff
@@ -0,0 +1,31 @@
+NWBFile:
+  experiment_description: CV experiments with ecog recording
+  experimenter: Jessie Liu
+  identifier: ADDME
+  institution: UC San Francisco
+  lab: Chang lab
+  session_description: ADDME
+  session_start_time: 2020-04-04 10:00:00
+  surgery: description of surgery
+Subject:
+  sex: Male
+  subject_id: I5
+  date_of_birth: 1980-04-04 00:00:00
+Ecephys:
+  Device:
+    - name: AmygdalaDepth
+    - name: CingulateDepth
+    - name: HippocampusDepth
+  ElectrodeGroup:
+    - description: ADDME
+      device: AmygdalaDepth
+      location: Amygdala
+      name: AmygdalaDepth
+    - description: ADDME
+      device: CingulateDepth
+      location: Cingulate
+      name: group_CingulateDepth
+    - description: ADDME
+      device: HippocampusDepth
+      location: Hippocampus
+      name: group_HippocampusDepth
```

Comment:
let's take out the electrode group and device names from the yaml. I think that will be confusing for them since we will be getting that info from the TDT_elecs_all.mat file.

Reply:
done